apps/website/src/components/Card.astro

```diff
@@ -0,0 +1,81 @@
+---
+import React from 'react';
+import ComponentIcon from './ComponentIcon.astro';
+
+export type Props = {
+  url: string;
+  title: React.ReactNode;
+  description?: React.ReactNode;
+  image?: string;
+  className?: string;
+};
+
+const { url, title, description, image, className } = Astro.props;
```

Comment:
after forwarding all props, we need to assign `class` to `className` because `class` is a reserved keyword. also grab `...rest`
```suggestion
const { url, title, description, image, class: className, ...rest } = Astro.props;
```